packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -9,43 +10,64 @@ This page and functionality are under construction. See https://github.com/flutt
 
 ### Glossary
 
-**GC**: garbage collection.
+**GC**: garbage collection. The process of reclaiming memory that is no
+longer being used.
 
-**Memory Leak**: progressive usage of more and more memory by an application, for example repetedly creating and not disposing a listener.
+**Memory Leak**: progressive use of more and more memory by an application,
+for example, by repeatedly creating (but not disposing of) a listener.
 
-**Memory Bloat**: use of more memory than is necessary for optimal performance, for example using too large images or not canceling a stream.
+**Memory Bloat**: using more memory than is necessary for optimal performance,
+for example, by using overly large images or not canceling a stream.
 
-**Object's Retaining Path**: path from the object to a root object that prevents the object from being GCed.
+**Object's Retaining Path**: references to an object from a root object that
+prevents the object from being GCed.
 
 
 ### Leak Types
 
-To detect memory leaks, the tool uses the fact that, with proper memory management, a dart object disposal and GC events should happen sequentially, close to each other.
+To detect memory leaks, the tool assumes that, with proper memory management,
+an object's disposal and collection should happen sequentially, close to each other.
 
-By disposal and GC events, the tool detects different types of leaks:
+By monitoring disposal and GC events, the tool detects different types of leaks:
 
-**Not disposed, but GCed (not-disposed)**: a disposable object was GCed, without being disposed first. This means that the object's disposable content was allocating memory, after the object became not needed. To fix the leak, invoke `dispose()` when the object is not needed.
+**Not disposed of, but GCed (not-disposed)**: a disposable object was GCed,
+without being disposed first. This means that the object's disposable content
+is using memory after the object was no longer needed.
+To fix the leak, invoke `dispose()` to free up the unneeded object.
 
-**Disposed, but not GCed (not-GCed)**: an object was disposed, but not GCed after certain number of GC events. This means there is a retaining path that holds the object from being garbage collected, after the object became not needed. To fix the leak, after disposal assign to null all reachable references to the object:
+**Disposed of, but not GCed (not-GCed)**: an object was disposed,
+but not GCed after certain number of GC events. This means that
+a reference to the object is preventing it from being
+garbage collected after it's no longer needed.
+To fix the leak, after disposal assign all references
```

Comment:
Please, remove 'of'.

Reply:
Word 'reachable' is important here. Some objects can still reference the object, but if they are unreachable, it is ok, it will not create a problem, because all unreachable objects (those who do not have retaining path) are collected.

This article explains it: https://medium.com/flutter/flutter-dont-fear-the-garbage-collector-d69b3ff1ca30

Reply:
ok

Reply:
? Where should I add 'reachable'?